crates/ere-hosts/src/main.rs

```diff
@@ -142,37 +142,39 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     };
 
+    let corpuses = block_witness_gen.generate().await?;
+
     // Set to true once a zkvm has ran
     let mut ran_any = false;
 
     #[cfg(feature = "sp1")]
     {
         run_cargo_patch_command("sp1")?;
         let sp1_zkvm = new_sp1_zkvm(resource)?;
-        run_benchmark_ere("sp1", sp1_zkvm, action, &block_witness_gen).await?;
+        run_benchmark_ere("sp1", sp1_zkvm, action, &corpuses).await?;
```

Comment:
@kevaundray, ah I think this merge with the `master` branch is wrong.
It shouldn't have the `.await` since isn't async anymore.

I think this wasn't catch by the CI since prob we need to tune to enable all features or at least 1?

Reply:
Ah sorry, yeah you're right